contracts/.env.template

```diff
@@ -1,2 +1,4 @@
+
 INFURA_ID=yourInfuraApiKey
 MNEMONIC=test test test test test test test test test test test junk
+FLEEK_STORAGE_API_KEY="yourFleekApiKey"
```

Comment:
We should probably mention in the Readme (the main readme) that you need a Fleek API key as we do for the other two services.

Reply:
Done in 6575ff8b8dc9d8d0c86e70d91018a98e63174cf8. When the endpoints we're using are publicly available we can expand this section.